packetbeat/config/config.go

```diff
@@ -59,64 +65,72 @@ type Icmp struct {
 	TransactionTimeout *int  `yaml:"transaction_timeout"`
 }
 
-type Dns struct {
-	ProtocolCommon      `yaml:",inline"`
-	Include_authorities *bool
-	Include_additionals *bool
+// DNS holds DNS specific configuration information
+type DNS struct {
+	ProtocolCommon     `yaml:",inline"`
+	IncludeAuthorities *bool
```

Comment:
As the yaml parser does some automatic name magic, you have to add now `yaml:",include_authorities"` (backticks are missing because of Github formatting). Same for the line below.